init-tools.sh

```diff
@@ -1,25 +1,27 @@
 #!/usr/bin/env bash
 
+set -u
```

Comment:
Did this come from an existing init-tools.sh change? It does look like it'll work (including in the scripts called by this script) but I'm wondering if it's been put in a repo somewhere already to get a "real" verification.

Reply:
Yes I'm interested in understanding why this is here now as well. Do keep in mind that most repo's "source" in this script so it isn't ran directly in most cases and so it is inheriting the set behavior of the outer script.

Reply:
If this is meant to be sourced, perhaps we should do a `set +u` at the end to prevent any unexpected behaviour change from leaking into the caller?

Reply:
That has the opposite problem of disabling `u` if the caller had it enabled. I think the most reasonable way to temporarily set this is with a subshell. (I tried this out in a workaround that ended up not being used.)

Reply:
I'll remove this.  After discussing with @dagood, this doesn't make sense.